epinar/assignment5/unit_tests

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
+
```

Comment:
It should be better to name this file "unit_tests.py" or like this
Lack of the extension is confusing

Reply:
Oops i lost the extension while moving files to manage the commit issues, will fix it 👍